src/cmd/builtin/heaptrace.c

```diff
@@ -76,37 +86,47 @@ int cmd_heaptrace (optparse_t *p, int ac, char *av[])
     return (0);
 }
 
-static struct optparse_subcommand heaptrace_subcmds[] = {
-    { "start",
-      "FILENAME",
-      "start heap profiling, sending output to FILENAME",
-      internal_heaptrace_start,
-      0,
-      NULL,
-    },
-    { "stop",
-      NULL,
-      "stop heap profiling",
-      internal_heaptrace_stop,
-      0,
-      NULL,
-    },
-    { "dump",
-      "REASON",
-      "dump heap profile",
-      internal_heaptrace_dump,
-      0,
-      NULL,
-    },
-    OPTPARSE_SUBCMD_END
-};
+static struct optparse_subcommand heaptrace_subcmds[] =
+    {{
+         "start",
+         "FILENAME",
+         "start heap profiling, "
```

Comment:
Since clang-format seems happy to break lines but not merge them, I think if I undo the `format` commits, squash the changes to the `.clang-format` file, and re-run `format`, this string won't be broken across lines like this.